libs/response-lib.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Handle a successfuly API call return case.
```

Comment:
If more Lambdas existed across git repos, how would you recommend re-using these common helpers?

Reply:
We could create a node package, that would be depended on across the repos to give a uniform integration with DynamoDB.  These helpers were just a basic approach I took in this assignment.